pkg/volume/plugins.go

```diff
@@ -201,6 +202,14 @@ type AttachableVolumePlugin interface {
 	GetDeviceMountRefs(deviceMountPath string) ([]string, error)
 }
 
+// ExpandableVolumePlugin is an extended interface of VolumePlugin and is used for volumes that can be
+// expanded
+type ExpandableVolumePlugin interface {
+	VolumePlugin
+	ExpandVolumeDevice(spec *Spec, newSize resource.Quantity, oldSize resource.Quantity) (resource.Quantity, error)
```

Comment:
Does it return the new size of the volume after the operation? These methods should have comments.

Reply:
What's the unit for these sizes?

Reply:
Yes it does return new size after volume operation.  I agree about comments. Added a github issue for myself so as I don't foget - https://github.com/kubernetes/kubernetes/issues/56177